addon/templates/components/nypr-brick-item.hbs

```diff
@@ -26,12 +26,12 @@
         type='blue-minion'
         state=(if (eq (string pk) audio.currentAudio.id) audio.playState)
         itemPK=item.id
-        itemTitle=item.attributes.title}}
+        itemTitle=(if item.attributes.shortTitle item.attributes.shortTitle item.attributes.title)}}
         Listen {{item.attributes.audioDurationReadable}}
       {{/listen-button}}
     {{/if}}
   </div>
   {{#link-to 'story' item.attributes.slug class="item__link--hidden"}}
-    {{item.attributes.title}}
+    {{if item.attributes.shortTitle item.attributes.shortTitle item.attributes.title}}
```

Comment:
What about a computed property instead of repeating this long conditional several times?